Tools/WebKitTestRunner/mac/main.mm

```diff
@@ -53,12 +53,15 @@ static void setDefaultsToConsistentValuesForTesting()
 
 static void disableAppNapInUIProcess()
 {
-    NSActivityOptions options = (NSActivityUserInitiatedAllowingIdleSystemSleep | NSActivityLatencyCritical) & ~(NSActivitySuddenTerminationDisabled | NSActivityAutomaticTerminationDisabled);
-    static NeverDestroyed<RetainPtr<id>> assertion = [[NSProcessInfo processInfo] beginActivityWithOptions:options reason:@"WebKitTestRunner should not be subject to process suppression"];
+    static NeverDestroyed<RetainPtr<id>> assertion;
+    static dispatch_once_t onceToken;
+    dispatch_once(&onceToken, ^{
+        NSActivityOptions options = (NSActivityUserInitiatedAllowingIdleSystemSleep | NSActivityLatencyCritical) & ~(NSActivitySuddenTerminationDisabled | NSActivityAutomaticTerminationDisabled);
+        assertion.get() = retainPtr([[NSProcessInfo processInfo] beginActivityWithOptions:options reason:@"WebKitTestRunner should not be subject to process suppression"]);
```

Comment:
Is the call to retainPtr() required here? 'assertion' is declared with a concrete type, so I don't think we need help with template argument deduction.

Reply:
It is not strictly necessary.  I was thinking we did this explicitly in most cases, but I think it's only when we want to use `auto` for a new variable declaration so we don't have to type out the whole `RetainPtr<>` type.

I'll remove it.